types/activex-excel/index.d.ts

```diff
@@ -11,38 +11,53 @@
 /// <reference types="activex-dao" />
 
 declare namespace Excel {
-    type AddSheetResult<T> =
-        T extends XlSheetType.xlDialogSheet ? DialogSheet :
-        T extends XlSheetType.xlChart ? Chart :
-        T extends XlSheetType ? Worksheet :
-        T extends undefined ? Sheet :
-        never;
-
-    type InternationalResult<T> =
-        T extends XlApplicationInternationalStrings ? string :
-        T extends XlApplicationInternationalNumbers ? number :
-        T extends XlApplicationInternationalBooleans ? boolean :
-        any;
```

Comment:
The intention of a conditional type might be clearer if formatted as above.

Reply:
There is some discussion about how to format the conditional operator in the prettier repository itself, but nothing we can change from here as it's not a tunable option.